Kata RPG/Kata RPG/Priest.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Priest.swift
+//  Kata RPG
+//
+//  Created by norm on 07/11/2020.
+//
+
+import Foundation
+
+
+class Priest: Character {
+
+        override func attack(target: Character){
+               print("\(self.name) is a priest and can't attack")
+            }
+
+
+    override func heal(target: Character) {
+                    if(self.lifeStatus == true){
+                        target.health += Int.random(in: 5...10)
```

Comment:
You should inject the random generator otherwise you won't be able to test that function properly

https://stackoverflow.com/questions/311807/unit-testing-with-functions-that-return-random-results/311829#311829